### DISCUSSION TITLE
Curating the Editor Experience: Tips & Tricks

### DISCUSSION THREAD

**Discussion by ndiego** [WordPress/developer-blog-content#262] [Topic Ideas]
The title is still TBD, but the idea of this article will be to showcase the many lesser known ways you can curate the Editing experience in WordPress. We already have a few articles on this topic that cover bigger topics like [client-side](https://developer.wordpress.org/news/2023/05/24/curating-the-editor-experience-with-client-side-filters/) and [server-side](https://developer.wordpress.org/news/2023/07/05/how-to-modify-theme-json-data-using-server-side-filters/) filtering, but I think it would be fun to showcase ~10 real-world examples and then directs the reader back to the official documentation and/or other related articles. 

This topic idea corresponds with [ongoing work](https://github.com/WordPress/gutenberg/issues/56170) to improve the [Curating the Editor Experience](https://developer.wordpress.org/block-editor/how-to-guides/curating-the-editor-experience/) section of the Block Editor Handbook.

**Comment by flexseth** [comment]
How about a media gallery that's draggable from the admin, showcases thumbnails with light boxes on the front-end

Make the admin as minimalistic as possible. Almost like - just the media gallery - in the post editor

for the UI: Only a canvas with the existing gallery. User can drag around to reorder, add a photo or remove photos in the gallery in the user interface. Created to simplify the admin for gallery pages

Remove the sidebar, put the controls in a meaningful space in the gallery, preview button

Showcases how to hide UI by role type using JavaScript

**Comment by bph** [comment]
The topic was approved. Please continue the conversations on the issue

- #276

**Comment by ndiego** [comment]
Post has been published: https://developer.wordpress.org/news/2024/07/04/15-ways-to-curate-the-wordpress-editing-experience/
